root/documentation/variation.conf

```diff
@@ -176,9 +176,19 @@
       </species>
       <fields>
         type=String
-        description=Comma-separated list of identifiers/notations to include from the following types: id (variant ID), hgvsg (HGVS genomic), hgvsc (HGVS coding), hgvsp (HGVS protein), spdi (SPDI genomic), vcf_string (VCF format)
+        description=Comma-separated list of identifiers/notations to include from the following types: id (variant ID), hgvsg (HGVS genomic), hgvsc (HGVS coding), hgvsp (HGVS protein), spdi (SPDI genomic)
         default=id,hgvsg,hgvsc,hgvsp,spdi
       </fields>
+      <vcf_string>
+        type=String
```

Comment:
Having looked at how these options are used in the test case - would it be better to describe the type of these options as `Boolean(0,1)` ?

Reply:
It makes more sense to be a boolean, thanks for pointing that out.